user/details/style.css

```diff
@@ -278,3 +278,35 @@ footer p a {
     height: 20px;
   }
 }
+
+.accordion-disabled {
+  opacity: 0.5;
+  cursor: none;
+}
+
+.accordion-icon {
+  width: 50%;
+}
+
+.tool-parent {
+  position: relative;
+}
```

Comment:
if this is no longer used can we remove it

Reply:
changed